src/components/Jobs.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <v-container>
+    <v-layout justify-center>
+      <v-flex md9>
+        <h1 class="mb-2">Jobs</h1>
+        <v-data-table
+          :headers="headers"
+          :items="jobs"
+          :pagination.sync="pagination"
+          must-sort
+          class="elevation-8"
+        >
+          <template v-slot:items="props">
+            <td>{{ props.item.product_name }}</td>
+            <td>
+              {{ organism(model(props.item.model_id).organism_id).name }}
+            </td>
+            <td>{{ model(props.item.model_id).name }}</td>
+            <td>
+              {{ props.item.status }}
+              <v-progress-circular
+                indeterminate
+                color="primary"
+                class="ml-2"
+                :width="3"
+                :size="25"
+                v-if="
+                  props.item.status === 'STARTED' ||
+                    props.item.status === 'PENDING'
+                "
+              ></v-progress-circular>
+            </td>
+            <td>{{ props.item.created | moment("D MMM YYYY, HH:mm") }}</td>
+            <td>
+              <router-link
+                :to="{ name: 'jobDetails', params: { id: props.item.id } }"
+                class="link"
+                ><v-btn color="primary" flat>Details</v-btn></router-link
+              >
+            </td>
+          </template>
+        </v-data-table>
+      </v-flex>
+    </v-layout>
+  </v-container>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+import { mapGetters } from "vuex";
+
+Vue.use(require("vue-moment"));
```

Comment:
Nice addition - let's add it globally instead?

Reply:
Of course, will do it